Portal/test/Datahub.Infrastructure.UnitTests/Services/WorkspaceBudgetManagementServiceTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿using Azure.Identity;
+using Azure.ResourceManager;
+using Datahub.Core.Model.Datahub;
+using Datahub.Infrastructure.Services.Cost;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using static Datahub.Infrastructure.UnitTests.Testing;
+using NSubstitute;
+
+namespace Datahub.Infrastructure.UnitTests.Services
+{
+    [TestFixture]
+    [Ignore("Missing configuration")]
+    public class WorkspaceBudgetManagementServiceTests
+    {
+        private WorkspaceBudgetManagementService _sut;
```

Comment:
Just noticed this is called `_sut`? What's that for? lol

Reply:
I'm not sure what you mean, `sut` = system under test

Reply:
Also note this is legacy/unused